lib/modules/manager/gradle/parser/common.spec.ts

```diff
@@ -0,0 +1,105 @@
+import type { Ctx } from '../types';
+import {
+  cleanupTempVars,
+  coalesceVariable,
+  interpolateString,
+  loadFromTokenMap,
+  storeInTokenMap,
+  storeVarToken,
+  stripReservedPrefixFromKeyTokens,
+} from './common';
+
+describe('modules/manager/gradle/parser/common', () => {
+  let ctx: Ctx;
+  const token = {} as never;
```

Comment:
shouldn't this be a `lexer.Token` ?

Reply:
I assumed it would be fine to use this dummy value here, since the token's actual content is not relevant for these tests. If this is considered "bad practice" or there are other reasons why not to do it, please let me know and I'll mock a proper `lexer.token`.

Reply:
yes, we should avoid untyped values, even in tests. you can use our partial helper to create an empty mock.